linux/setup/reconfig_existing_guest.yml

```diff
@@ -37,7 +53,7 @@
     - name: "Install 'sudo', 'gawk', 'python3-rpm', 'tar' on VMware Photon OS"
```

Comment:
'python3-rpm' can also be removed from the task name

Reply:
Fixed. Thanks!